pkg/agent/datamodel/helper.go

```diff
@@ -93,3 +93,19 @@ func IndentString(original string, spaces int) string {
 	}
 	return out.String()
 }
+
+func processContainerImageTag(downloadURL string) (string, error) {
```

Comment:
sorry I know I recommended this before, but can we do `processContainerImage` instead since these technically aren't tags, rather wild-carded repos